benchmarks/benchmark_latency.py

```diff
@@ -108,43 +142,51 @@ def run_to_completion(profile_dir: Optional[str] = None):
         }
         with open(args.output_json, "w") as f:
             json.dump(results, f, indent=4)
+        save_to_pytorch_benchmark_format(args, results)
 
 
-if __name__ == '__main__':
+if __name__ == "__main__":
```

Comment:
Are these changed by linter automatically?

Reply:
Yeah, they comes from the pre-commit ruff linter